src/main/java/com/univocity/parsers/containers/Csv.java

```diff
@@ -0,0 +1,48 @@
+package com.univocity.parsers.containers;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * @author naveen.kasthuri
+ *
+ * A container that maps column indices to thier names. This avoids having duplicate arrays of
+ * key-value pairs where the keys are the same for every row in the csv file.
+ *
+ * This is used by the Row object and the RowListProcessor. Should not be used otherwise.
+ */
+public class Csv {
+  // Contains a mapping between row index and column
+  private final Map<String, Integer> columnIndexMap;
```

Comment:
This class won't work with files that don't have headers. Maybe you'd need to have another version of this class that uses  an `EnumMap` when an enumeration is provided.